docs/index.rst

```diff
@@ -47,8 +47,14 @@ This will add a `migrations` folder to your application. The contents of this fo
 
 You can then generate an initial migration::
 
-    $ flask db migrate
-    
+    $ flask db migrate #-m "You can leave optional message."
```

Comment:
I prefer that you only show the version of this command with a message. Adding the message but commented out is confusing, since it has no effect. So please remove the `#` and change the message to something that is appropriate for this example, such as "Initial migration".

Reply:
Ok!